### DISCUSSION TITLE
v1.0.0-beta7

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1057] [General]
新增: 在 `windows` 下双击打开使用弹窗提醒. by @Ink-33 
新增: `/get_unidirectional_friend_list` API. 用于获取单向好友列表.  [文档](https://github.com/Mrs4s/go-cqhttp/blob/master/docs/cqhttp.md#%E8%8E%B7%E5%8F%96%E5%8D%95%E5%90%91%E5%A5%BD%E5%8F%8B%E5%88%97%E8%A1%A8) by @Mrs4s 
新增: `/delete_unidirectional_friend_list` API. 用于删除单向好友. [文档](https://github.com/Mrs4s/go-cqhttp/blob/master/docs/cqhttp.md#%E5%88%A0%E9%99%A4%E5%8D%95%E5%90%91%E5%A5%BD%E5%8F%8B) by @Mrs4s 
新增: `[CQ:image]` 类型增加 `subType` 字段, 用于获取或设置图片子类型. [文档](https://github.com/Mrs4s/go-cqhttp/blob/master/docs/cqhttp.md#%E5%9B%BE%E7%89%87) by @Mrs4s 
新增: 长轮询服务支持 `timeout` 参数了. by @wdvxdr1123 
新增: 支持了 `windows/arm64` 架构. by @wdvxdr1123 

修复: 私聊分享音乐无法发送的问题. by @wdvxdr1123 
修复: `/.handle_quick_operation` 无法处理非字符串 `flag` 参数的问题. by @Mrs4s 
修复: `/send_group_forward_msg ` 无法从非 `json` 请求中获取参数的问题. by @Mrs4s 
修复: `log-aging` 配置设置为 `0` 仍然会删除日志的问题. by @Mrs4s 
修复: `/mark_msg_as_read` 参数类型错误的问题. by @Yukari316 
修复: 环境变量配置初始化时会 `panic` 的问题. by @502647092 
修复: 使用 `address.txt` 设置自定义服务器地址时会 `panic` 的问题. by @wdvxdr1123 
修复: 由于在错误的时机加载环境变量配置, 导致自动重连被关闭的问题. by @Mrs4s 
修复: 部分情况下自动重连会出错的问题. by @Mrs4s 
修复: 无法在收到第一条消息前向单向好友发送消息的问题. by @Mrs4s 
修复: 使用 `云函数 Lambda` 时调用出错无任何返回的问题. by @wdvxdr1123 

优化: 在 `amd64` 的CPU架构下使用 `avx2` 指令集对 `base64` 参数进行解码. by @wdvxdr1123 
优化: 构建环境升级到 `go1.17`. by @wdvxdr1123 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-beta7'>v1.0.0-beta7</a>.</em>

**Comment by xiaobaihu66** [comment]
[2023-01-24 19:03:44] [ERROR]: Protocol -> panic on decoder PttCenterSvr.ShortVideoDownReq : runtime error: index out of range [0] with length 0
goroutine 522 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x64
github.com/Mrs4s/MiraiGo/client.(*QQClient).netLoop.func1.1()
        github.com/Mrs4s/MiraiGo@v0.0.0-20221202060717-4658474c60dd/client/network.go:335 +0x48
panic({0x609660, 0x40000d8288})
        runtime/panic.go:884 +0x20c
github.com/Mrs4s/MiraiGo/client.decodePttShortVideoDownResponse(0x4000414180?, 0x24?, {0x4000414180, 0x24, 0x24})
        github.com/Mrs4s/MiraiGo@v0.0.0-20221202060717-4658474c60dd/client/ptt.go:337 +0x15c
github.com/Mrs4s/MiraiGo/client.(*QQClient).netLoop.func1(0x40001d2780)
        github.com/Mrs4s/MiraiGo@v0.0.0-20221202060717-4658474c60dd/client/network.go:346 +0x184
created by github.com/Mrs4s/MiraiGo/client.(*QQClient).netLoop
        github.com/Mrs4s/MiraiGo@v0.0.0-20221202060717-4658474c60dd/client/network.go:332 +0x4e4
 
[2023-01-24 19:03:44] [ERROR]: 出现错误 packet decode error: PttCenterSvr.ShortVideoDownReq - runtime error: index out of range [0] with length 0. 详细信息已转储至文件 dumps/1674558224.dump 请连同日志提交给开发者处理
